<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>eso50cm telescope console</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindows"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="saveAsAction"/>
   <addaction name="stopTelescopeAction"/>
   <addaction name="parkTelescopeAction"/>
   <addaction name="parkTelescopeCap"/>
   <addaction name="gotoTargetAction"/>
   <addaction name="trackingAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockProcess">
   <property name="windowTitle">
    <string>system</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockLogger">
   <property name="windowTitle">
    <string>logger</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QDockWidget" name="dockWebcam">
   <property name="windowTitle">
    <string>webcam</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <widget class="QDockWidget" name="dockOffset">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>300</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>offset</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4"/>
  </widget>
  <widget class="QDockWidget" name="dockEncoder">
   <property name="windowTitle">
    <string>encoders</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5"/>
  </widget>
  <action name="saveAsAction">
   <property name="icon">
    <iconset resource="Console.qrc">
     <normaloff>:/eso50cm/Cosole/save.png</normaloff>:/eso50cm/Cosole/save.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save As</string>
   </property>
   <property name="toolTip">
    <string>Save As ...</string>
   </property>
  </action>
  <action name="gotoTargetAction">
   <property name="icon">
    <iconset resource="Console.qrc">
     <normaloff>:/eso50cm/Cosole/goto_target.png</normaloff>:/eso50cm/Cosole/goto_target.png</iconset>
   </property>
   <property name="text">
    <string>Goto Target</string>
   </property>
   <property name="toolTip">
    <string>goto target</string>
   </property>
  </action>
  <action name="stopTelescopeAction">
   <property name="icon">
    <iconset resource="Console.qrc">
     <normaloff>:/eso50cm/Cosole/stop_telescope.png</normaloff>:/eso50cm/Cosole/stop_telescope.png</iconset>
   </property>
   <property name="text">
    <string>Stop Telescope</string>
   </property>
   <property name="toolTip">
    <string>stop telescope</string>
   </property>
  </action>
  <action name="parkTelescopeAction">
   <property name="icon">
    <iconset resource="Console.qrc">
     <normaloff>:/eso50cm/Cosole/park_telescope.png</normaloff>:/eso50cm/Cosole/park_telescope.png</iconset>
   </property>
   <property name="text">
    <string>Park Telescope</string>
   </property>
   <property name="toolTip">
    <string>park telescope</string>
   </property>
  </action>
  <action name="parkTelescopeCap">
   <property name="icon">
    <iconset resource="Console.qrc">
     <normaloff>:/eso50cm/Cosole/park_cap.png</normaloff>:/eso50cm/Cosole/park_cap.png</iconset>
   </property>
   <property name="text">
    <string>Park Telescope Cap</string>
   </property>
   <property name="toolTip">
    <string>park telescope cap</string>
   </property>
  </action>
  <action name="trackingAction">
   <property name="icon">
    <iconset resource="Console.qrc">
     <normaloff>:/eso50cm/Cosole/tracking.png</normaloff>:/eso50cm/Cosole/tracking.png</iconset>
   </property>
   <property name="text">
    <string>Tracking</string>
   </property>
   <property name="toolTip">
    <string>set tracking</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Console.qrc"/>
 </resources>
 <connections/>
</ui>
